src/pages/index.tsx

```diff
@@ -1,41 +1,371 @@
+	'dropbox.svg',
+	'instagram.svg',
+	'facebook.svg',
+	'onedrive.svg',
+];
+
+const frameworks = [
+	{ name: 'React', Icon: IconReact, code: reactCode },
+	{ name: 'Vue', Icon: IconVue, code: vueCode },
+	{ name: 'Svelte', Icon: IconSvelte, code: svelteCode },
+	{ name: 'Angular', Icon: IconAngular, code: angularCode },
+];
 
 export default function Home(): JSX.Element {
-	const { siteConfig } = useDocusaurusContext();
+	const [framework, setFramework] = useState(frameworks[0].name);
+
 	return (
-		<Layout
-			title={`Hello from ${siteConfig.title}`}
-			description="Description will go into a meta tag in <head />"
-		>
-			<HomepageHeader />
-			<main>
-				<HomepageFeatures />
+		<Layout description="Description will go into a meta tag in <head />">
+			<header className={styles.header}>
+				<h1>Sleek, modular open source JavaScript file uploader</h1>
+
+				<p>
+					Uppy fetches files locally and from remote places like Dropbox or
+					Instagram. With its seamless integration, reliability and ease of use,
+					Uppy is truly your best friend in file uploading.
+				</p>
+
+				<Link className={styles.button} to="/docs/quick-start">
+					Get started
+				</Link>
+
+				<figure className={styles.quote}>
+					<blockquote cite="https://stackshare.io/posts/top-developer-tools-2017">
+						<p>“Top 10 tools of the year”</p>
+					</blockquote>
+					<figcaption>
+						<a
+							href="https://stackshare.io/posts/top-developer-tools-2017"
+							rel="noopener"
+							target="_blank"
+						>
+							Stackshare
+						</a>
+					</figcaption>
+				</figure>
+
+				<figure className={styles.quote}>
+					<blockquote cite="https://books.producthunt.com/bestof2017">
+						<p>“The best product launches”</p>
+					</blockquote>
+					<figcaption>
+						<a
+							href="https://books.producthunt.com/bestof2017"
+							rel="noopener"
+							target="_blank"
+						>
+							Product Hunt
+						</a>
+					</figcaption>
+				</figure>
+
+				<figure className={styles.quote}>
+					<blockquote cite="https://twitter.com/smashingmag/status/1097870169043546112">
+						<p>“Soooo useful”</p>
+					</blockquote>
+					<figcaption>
+						<a
+							href="https://twitter.com/smashingmag/status/1097870169043546112"
+							rel="noopener"
+							target="_blank"
+						>
+							Smashing Magazine
+						</a>
+					</figcaption>
+				</figure>
+			</header>
+
+			<main className={styles.main}>
+				<section className={styles['section-dashboard']}>
+					<h2>
+						The all you need Dashboard — powerful, responsive, and pluggable.
+					</h2>
+					<p>
+						Add files from remote sources, edit images, generate thumbnails, and
+						more.
+					</p>
+					<div className={styles.dashboard}>
+						<BrowserOnly>
+							{() => {
+								const uppy = new Uppy().use(Webcam);
+								return (
+									<Dashboard uppy={uppy} height={300} plugins={['Webcam']} />
+								);
+							}}
+						</BrowserOnly>
+						<CodeBlock language="js" showLineNumbers>
```

Comment:
```suggestion
						<CodeBlock language="js">
```

Propose to remove `showLineNumbers` to save horizontal space — currently the code block has horizontal scrollbar even on desktop.

Reply:
+1, I think those can safely be ignored on the homepage.